life-letters/modernized/06-chapter-5.adoc

```diff
@@ -701,12 +701,14 @@ London, Seventh month 15th, 1739.
 
 I received your acceptable letter along with dear father`'s,
 both which came very seasonably-- I was wanting to hear from you both.
-I wonder not at the tenderness you expresses in yours; while he was here,
+I wonder not at the tenderness you express in yours; while he was here,
 my heart was filled with thankfulness, not only for such a parent,
 but also that I could see some marks of that treasure of goodness,
 which it has pleased Divine love to place and treasure up in his soul.
 The wisdom of his conduct, his uprightness and humility, often affect me,
 and make me wish, "`Oh, __si ut alter ab illo?__`"
+//Would it be better to add the English translation here? 
```

Comment:
Yes, it would... I just couldn't find a good translation of this expression into English, and I didn't want to guess. :(